cli.js

```diff
@@ -3,10 +3,12 @@
 /**
  * Parsing:
  *
- * $ debuguy parse <source_dir>
- * $ debuguy parse <source_dir> <destination_dir>
- * $ debuguy parse -r <source_dir>
- * $ debuguy parse -r <source_dir> <destination_dir>
+ * $ debuguy parse <source_dir> [options]
+ * $ debuguy parse <source_dir> <destination_dir> [options]
```

Comment:
I suggest we follow the general consensus in using command line options, such as git. Making the sequence of the command line options different from such popular tool will confuse users.

In git, 'git checkout -b test_branch' works while 'git checkout test_branch -b' doesn't.